tests/testsuite/doc.rs

```diff
@@ -1518,9 +1545,16 @@ fn doc_workspace_open_different_library_and_package_names() {
 
     p.cargo("doc --open")
         .env("BROWSER", tools::echo())
-        .with_stderr_contains("[..] Documenting foo v0.1.0 ([..])")
-        .with_stderr_contains("[..] [CWD]/target/doc/foolib/index.html")
-        .with_stdout_contains("[CWD]/target/doc/foolib/index.html")
+        .with_stderr_data(str![[r#"
+[DOCUMENTING] foo v0.1.0 ([ROOT]/foo/foo)
+[FINISHED] `dev` profile [unoptimized + debuginfo] target(s) in [ELAPSED]s
+     Opening [ROOT]/foo/target/doc/foolib/index.html
```

Comment:
Hmm… Looks like we don't have redaction for `Opening`.

Reply:
Thanks, it has been added.